### PR TITLE
update rename locals to false

### PR DESCRIPTION
**Pull request by meaghanlewis**

### CODE REVIEW

File: TestWebAppProject/bundleconfig.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     // Optionally specify minification options
     "minify": {
       "enabled": true,
-      "renameLocals": true
+      "renameLocals": false
     },
     // Optionally generate .map file
     "sourceMap": false
```
Comment (meaghanlewis) [inline TestWebAppProject/bundleconfig.json:22]: comment on line 22

File: TestWebAppProject/bundleconfig.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
     // Optionally specify minification options
     "minify": {
       "enabled": true,
-      "renameLocals": true
+      "renameLocals": false
```
Comment (meaghanlewis) [inline TestWebAppProject/bundleconfig.json:19]: comment on line 19

File: TestWebAppProject/bundleconfig.json [hunk 3]
```diff
@@ -16,7 +16,7 @@
...
     },
```
Comment (meaghanlewis) [inline TestWebAppProject/bundleconfig.json:20]: }

Comment (meaghanlewis) [reply TestWebAppProject/bundleconfig.json:20]: replying

File: TestWebAppProject/bundleconfig.json [hunk 4]
```diff
@@ -16,7 +16,7 @@
...
     // Optionally generate .map file
```
Comment (meaghanlewis) [inline TestWebAppProject/bundleconfig.json:21]: comment here

**Review by meaghanlewis-tester — APPROVED** [review]
LGTM!

**Review by meaghanlewis-tester — CHANGES_REQUESTED** [review]
can you change the thing?

File: TestWebAppProject/bundleconfig.json [hunk 5]
```diff
@@ -16,7 +16,7 @@
     // Optionally specify minification options
     "minify": {
       "enabled": true,
```
Comment (meaghanlewis) [inline TestWebAppProject/bundleconfig.json:18]: something

File: TestWebAppProject/bundleconfig.json (hunk 2 above)
Comment (meaghanlewis) [reply TestWebAppProject/bundleconfig.json:19]: },
    // Optionally generate .map file
    "sourceMap": false
  }

Comment (meaghanlewis) [reply TestWebAppProject/bundleconfig.json:19]: https://github.com/meaghanlewis/TestWebAppProject/issues/5

Comment (meaghanlewis) [reply TestWebAppProject/bundleconfig.json:19]: https://media2.giphy.com/media/OODXXtfUiUmSdi3ExL/giphy.gif

Comment (meaghanlewis) [reply TestWebAppProject/bundleconfig.json:19]: img

Comment (meaghanlewis) [reply TestWebAppProject/bundleconfig.json:19]: :)

Comment (meaghanlewis) [reply TestWebAppProject/bundleconfig.json:19]: 👍

Comment (meaghanlewis) [reply TestWebAppProject/bundleconfig.json:19]: :thumbs_up:

Comment (meaghanlewis) [reply TestWebAppProject/bundleconfig.json:19]: :thumbsup:

File: TestWebAppProject/bundleconfig.json (hunk 1 above)
Comment (meaghanlewis) [reply TestWebAppProject/bundleconfig.json:22]: https://github.com/meaghanlewis/TestWebAppProject/issues/5

Comment (meaghanlewis) [reply TestWebAppProject/bundleconfig.json:22]: {
    "outputFileName": "wwwroot/js/site.min.js",
    "inputFiles": [
      "wwwroot/js/site.js"
    ],
    // Optionally specify minification options
    "minify": {
      "enabled": true,
      "renameLocals": true
    },
    // Optionally generate .map file
    "sourceMap": false
  }

File: TestWebAppProject/bundleconfig.json (hunk 5 above)
Comment (meaghanlewis) [reply TestWebAppProject/bundleconfig.json:18]: start review

File: TestWebAppProject/bundleconfig.json (hunk 2 above)
Comment (meaghanlewis) [reply TestWebAppProject/bundleconfig.json:19]: finish

File: TestWebAppProject/bundleconfig.json [hunk 6]
```diff
@@ -16,7 +16,7 @@
     // Optionally specify minification options
```
Comment (ghfvs-tester) [inline TestWebAppProject/bundleconfig.json:16]: minify?

Comment (ghfvs-tester) [reply TestWebAppProject/bundleconfig.json:16]: yep!

**Review by ghfvs-tester — APPROVED** [review]
LGTM

**Review by ghfvs-tester — COMMENTED** [review]
something something

File: TestWebAppProject/bundleconfig.json (hunk 4 above)
Comment (ghfvs-tester) [reply TestWebAppProject/bundleconfig.json:21]: not sure about this

Comment (ghfvs-tester) [reply TestWebAppProject/bundleconfig.json:21]: not sure about this

File: TestWebAppProject/bundleconfig.json [hunk 7]
```diff
@@ -16,7 +16,7 @@
...
     "minify": {
```
Comment (ghfvs-tester) [inline TestWebAppProject/bundleconfig.json:17]: yes

**Review by ghfvs-tester — CHANGES_REQUESTED** [review]

**Review by ghfvs-tester — APPROVED** [review]

**Pull request merged**